components/eamxx/src/physics/mam/eamxx_mam_microphysics_process_interface.cpp

```diff
@@ -197,6 +199,15 @@ void MAMMicrophysics::set_grids(
   // - extfrc: 3D instantaneous forcing rate [kg/m³/s]
   add_field<Computed>("mam4_external_forcing", scalar3d_extcnt, kg / m3 / s, grid_name);
 
+  // Register computed fields for tendencies due to gas phase chemistry
+  // - dvmr/dt: Tendencies for mixing ratios  [kg/kg/s]
+  diagnostic_gs_dvmrdt_ = m_params.get<bool>("output_mixing_ratio_tendency_due_to_gas_phase_chemistry", false);
+  if (diagnostic_gs_dvmrdt_)
+    add_field<Computed>("mixing_ratio_tendency_due_to_gas_phase_chemistry", scalar2d_gas_pcnst, kg / kg / s, grid_name);
```

Comment:
Does this also includes the tendency from microphysics (e.g., gas to particle conversion, which depends on the gas phase chemistry production)? If so, it would be useful to add some notes above.

Reply:
Probably you already know, this is `GS_*`, it is computed just after the gas phase chemistry, but before the aqueous chemistry (setsox). I am not sure if we convert gas to particles in `gas_phase_chemistry` call.

Reply:
Thanks for the note. I was not sure whether this is GS_*. The gas to particle conversion is considered in aerosol microphysics (amicphys), so it's not included here and the aerosol concentrations are not affected.